website/docs/releases/2023/v2023.5.md

```diff
@@ -37,8 +41,4616 @@ image:
+-   blueprints: ignore hidden files in discovery (#5472)
+-   blueprints: improve schema generation by including model schema (#5503)
+-   crypto: make name field unique to prevent double certs (#5406)
+-   events: always run policies for notification rules even if no group is selected (#5353)
+-   events: cleanse http query string in events (#5508)
+-   internal: ignore insecure TLS ciphers (#5483)
+-   lifecycle: migrate internal healthcheck to use go (#5322)
+-   outposts: make state more consistent (#5403)
+-   policies: clear app cache when writing user, groups, policies (#5371)
+-   policies: make policy engine modes consistent with database values (#5462)
+-   providers/ldap: correctly use pagination in search results in both modes (#5492)
+-   providers/ldap: remove deprecated fields (#5154)
+-   providers/oauth2: use simpler charset for refresh tokens (#5502)
+-   providers/scim: correctly handle 404 by re-creating object (#5405)
+-   providers/scim: ensure scim group member isn't None (#5391)
+-   providers/scim: fix missing user/group filtering on SCIM direct save signals (#5473)
+-   providers/scim: improve compatibility (#5425)
+-   sources/oauth: add patreon type (#5452)
+-   stages/prompt: Add initial_data prompt field and ability to select a default choice for choice fields (#5095)
+-   web/admin: add notes for users and groups (#5459)
+-   web/admin: add toggle to hide deactivated users (#5419)
+-   web/admin: fix blueprint instance list without metadata or labels (#5296)
+-   web/admin: fix cert expiry coloring (#5354)
+-   web/admin: fix error when prompt previewing fails when loading the page (#5290)
+-   web/admin: fix file path setting not saved properly (#5463)
+-   web/admin: fix flow stage binding update failing (#5287)
+-   web/admin: fix outpost integration list (#5418)
+-   web/admin: fix radius view page not imported (#5450)
+-   web/admin: fix state issue after clearIcon/Background is used and for… (#5423)
+-   web/admin: remove redundant table grouping (#5343)
+-   web/admin: use radio for client type (#5499)
+-   web/flow: render prompt inputs without unsafeHTML (#5404)
+-   web: Fix label not clickable for checkbox and choice field in prompts (#5355)
+-   web: fix API browser error (#5402)
+-   web: fix loading text not being loaded (#5497)
 
 ## API Changes
 
-_Insert output of `make gen-diff` here_
+#### What's Changed
+
+---
+
+##### `GET` /crypto/certificatekeypairs/{kp_uuid}/
+
+###### Return Type:
+
+Changed response : **200 OK**
+
+-   Changed content type : `application/json`
+
+    New required properties:
+
+    -   `managed`
+
+    *   Changed property `managed` (string)
+        > Objects which are managed by authentik. These objects are created and updated automatically. This is flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

Comment:
```suggestion
        > Objects that are managed by authentik. These objects are created and updated automatically. This flag only indicates that an object can be overwritten by migrations. You can still modify the objects via the API, but expect changes to be overwritten in a later update.
```

Reply:
The source text for this is here https://github.com/goauthentik/authentik/blob/8ccdbdc370defc14c3907d3a24ce4941109df8c7/authentik/blueprints/models.py#L32, probably better to edit it there and re-generate the api changes